### PR TITLE
Updated code comments to portuguese

### PR DESCRIPTION
**Pull request by CubeSkyy**
- [X] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `master` branch of freeCodeCamp.
- [X] None of my changes are plagiarized from another source without proper attribution.
- [X] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: guide/portuguese/algorithms/graph-algorithms/dijkstra/index.md [hunk 1]
```diff
@@ -48,27 +48,27 @@ import java.util.*;
...
-        // Iterate over neighbours of u and update their distances 
+        // Itera sobre os visinhos de u e atualiza as suas distancias
         while(iter.hasNext()) { 
         Pair<Integer> y = iter.next(); 
         int v = y.first; 
         int weight = y.second; 
-        // Check if vertex v is not visited 
-        // If new path through u offers less cost then update distance array and add to pq 
+        // Verifica se vertice v não foi visitado
+        // Se o novo caminho u oferece menor custo, actualiza o vector de distancias e adiciona a pq
```
Comment (jefferson2z) [inline guide/portuguese/algorithms/graph-algorithms/dijkstra/index.md]: traduzir palavra vector

File: guide/portuguese/algorithms/graph-algorithms/dijkstra/index.md [hunk 2]
```diff
@@ -48,27 +48,27 @@ import java.util.*;
     } 
     } 
  
-    // Calculates shortest path to each vertex from source and returns the distance 
+    // Calcula o caminho mais curto para cada vertice apartir da fonte e retorna a distancia
```
Comment (jefferson2z) [inline guide/portuguese/algorithms/graph-algorithms/dijkstra/index.md]: acentos na palavra distância

File: guide/portuguese/algorithms/graph-algorithms/dijkstra/index.md [hunk 3]
```diff
@@ -48,27 +48,27 @@ import java.util.*;
...
     public int[] dijkstra(Graph g, int src) { 
-    int distance[] = new int[gn]; // shortest distance of each vertex from src 
+    int distance[] = new int[gn]; // Menor distancia de cada vertice da fonte
```
Comment (jefferson2z) [inline guide/portuguese/algorithms/graph-algorithms/dijkstra/index.md]: acento na palavra distância

File: guide/portuguese/algorithms/graph-algorithms/dijkstra/index.md [hunk 4]
```diff
@@ -48,27 +48,27 @@ import java.util.*;
...
     boolean visited[] = new boolean[gn]; // vertex is visited or not 
     Arrays.fill(distance, Integer.MAX_VALUE); 
     Arrays.fill(visited, false); 
     PriorityQueue<Pair<Integer>> pq = new PriorityQueue<>(100, new PairComparator()); 
         pq.add(new Pair<Integer>(src, 0)); 
     distance[src] = 0; 
     while(!pq.isEmpty()) { 
-        Pair<Integer> x = pq.remove(); // Extract vertex with shortest distance from src 
+        Pair<Integer> x = pq.remove(); // Retira o vertice com menor distancia á fonte
```
Comment (jefferson2z) [inline guide/portuguese/algorithms/graph-algorithms/dijkstra/index.md]: acento na palavra vértice

Comment (jefferson2z) [reply guide/portuguese/algorithms/graph-algorithms/dijkstra/index.md]: "á" deveria ser a craseado ,i.e., "à"

File: guide/portuguese/algorithms/graph-algorithms/dijkstra/index.md [hunk 5]
```diff
@@ -48,27 +48,27 @@ import java.util.*;
...
         int u = x.first; 
         visited[u] = true; 
         Iterator<Pair<Integer>> iter = g.adj[u].listIterator(); 
-        // Iterate over neighbours of u and update their distances 
+        // Itera sobre os visinhos de u e atualiza as suas distancias
```
Comment (jefferson2z) [inline guide/portuguese/algorithms/graph-algorithms/dijkstra/index.md]: acento na palavra distâncias

File: guide/portuguese/algorithms/graph-algorithms/dijkstra/index.md [hunk 6]
```diff
@@ -48,27 +48,27 @@ import java.util.*;
...
         while(iter.hasNext()) { 
         Pair<Integer> y = iter.next(); 
         int v = y.first; 
         int weight = y.second; 
-        // Check if vertex v is not visited 
-        // If new path through u offers less cost then update distance array and add to pq 
+        // Verifica se vertice v não foi visitado
```
Comment (jefferson2z) [inline guide/portuguese/algorithms/graph-algorithms/dijkstra/index.md]: acento na palavra vértice

**Review by jeffersonataide — CHANGES_REQUESTED** [review]
Muito boas as modificações, apenas de uma olhada nos detalhes que eu comentei antes.

**Comment by jefferson2z** [comment]
Thanks for your contribution and congratulations for your first PR merge to this repo! :smiley_cat:

**Review by jeffersonataide — APPROVED** [review]

**Pull request merged**